index.html

```diff
@@ -3330,6 +3331,48 @@ <h1>Artists</h1>
                     id="sOcMT" alt="sOcMT" label="sOcMT" class="statNumber statTotal">0</span></p>
         </div>
 
+        <div id="addCodeStats" alt="addCodeStats" label="addCodeStats" class="statContainer" style="display: none;">
+            <p id="addCodeStatTitle1" alt="addCodeStatTitle1" label="addCodeStatTitle1" class="statPortion" style="color: green; font-size: 1.2em">Add Code Effects</p>
+            <p id="stat+eff1" alt="stat+eff1" label="stat+eff1" class="statPortion singularity">Quarks:<span id="s+eff1" alt="s+eff1" label="s+eff1" class="statNumber"></span></p>
```

Comment:
I think I need to remove the `singularity` bit on most of these; the reason for that part of the classlist _just_ clicked for me.

While working on a second stats page for global speed, since that's been requested a bit.